jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -93,7 +93,7 @@ public CRIUSupport(Path imageDir) {
 	}
 
 	/**
-	 * Queries if CRIU support is enabled and j9criu29 library has been loaded.
+	 * Queries if CRIU support is enabled.
```

Comment:
this needs to call `isCRIUSupportEnabledAndNativeLoaded`

Reply:
`isCRIUSupportEnabledAndNativeLoaded()` call is added.